test_debs/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e -o pipefail -x
+
+if [ -z "$IMAGE_TAG" ]; then
+    echo "IMAGE_TAG env needs to be set"
+    exit 1
+fi
+
+if [ -z "$1" ]; then
+    echo "first argument needs to be set - IMAGEMAGICK_VERSION"
+    exit 1
+fi
+
+IMAGEMAGICK_VERSION=$1
+
+echo "Testing package expected dependencies"
+dpkg-deb -f imagemagick-alfresco_${IMAGEMAGICK_VERSION}_amd64.deb  Depends | grep -qEv 'libcdt|libcgraph|libgvc|libgs9'
```

Comment:
I don't thikn we need that as we are going through the whole nstallation process now. If any is missing it will fail.
That also avoids having to maintain lib names which may change between distros and versions